docs/static/gamejam/rules.md

```diff
@@ -1,54 +1,86 @@
-## Garden Jam
+## Garden Jam Winners!
 
-The theme for this jam is "gardens". In other words, we're looking for games about gardening or set in a garden-like environment. Some possible ways to interpret this theme:
+We're thrilled to announce the winners of the MakeCode Arcade Garden Jam! Before we get to the good stuff, we also want to thank everyone who participated--it was wonderful to see the creative and unique interpretations of the "garden" theme, and we hope you're proud of your work. We've all enjoyed playing your games so much! Check out the full gallery of entries at the bottom of this page.
```

Comment:
oops i actually have to replace these with real em-dashes, marked doesn't do the formatting. haha i think it looks better without but i also see them most in the context of books (vs newspapers/news sites), so that might be the divide!

Reply:
Ahh, that makes sense, my 2 years as one of my high school newspaper's editors coming back to bite me!

Reply:
haha well blog post is definitely newspaper-like so i think you're right in this case!